euslisp/common.l

```diff
@@ -73,5 +75,9 @@
         (assert-isinstance robot robot-model)
         (send robot joint-method-symbol :joint-angle angle-eusunit)))
     (send ri :angle-vector (send robot :angle-vector))
+    (if (not (null rarm-angle-stdunit))
```

Comment:
@Kanazawanaoaki
I think if there is no if-else condition, just when or unless is clear.
https://programgenjin.hatenablog.com/entry/2019/03/06/112513

In this case unless keyword fits well.